charts/testmachinery/charts/argo/templates/config.yaml

```diff
@@ -28,21 +28,24 @@ data:
         limits:
           cpu: "100m"
           memory: "150Mi"
-    {{ if .Values.argo.logging.links }}
+    {{ if .Values.argo.logging }}
     links:
-    {{ range .Values.argo.logging.links }}
+    {{-  include "getDefaultLoggingUrl" . | nindent 4 }}
+    {{- if .Values.argo.logging.additionallinks }}
+    {{- range .Values.argo.logging.additionallinks }}
     - name: {{ .name }}
       scope: {{ .scope }}
       url: {{ .url }}
-    {{ end }}
-    {{ end }}
+    {{- end }}
+    {{- end }}
+    {{- end }}
```

Comment:
is the the resulting yaml fine with argo if `argo.logging` is defined but not `argo.logging.links`?